examples/basecoin/app/app_test.go

```diff
@@ -127,10 +127,13 @@ func TestGenesis(t *testing.T) {
 	}
 	acc := &types.AppAccount{baseAcc, "foobart"}
 
-	genesisState := types.GenesisState{
-		Accounts: []*types.GenesisAccount{
+	genesisState := map[string]interface{}{
+		"accounts": []*types.GenesisAccount{
 			types.NewGenesisAccount(acc),
 		},
+		"cool": map[string]string{
+			"trend": "ice-cold",
+		},
```

Comment:
is there another way than using a map here? I know it's safe

Reply:
maybe fine - what are our options

Reply:
Other option - pass bytestring directly - means we'd have to use `printf` for variable parameters, though, seems ugly.